### PR TITLE
Updated templates to support resource limit variables

### PR DESCRIPTION
**Pull request by anaga670**
All the template files have the same three changes/blocks added, so if you could just check that one looks good then you shouldn't need to look at the rest

### CODE REVIEW

File: omar/openshift/templates/apps/omar-oms-app.json [hunk 1]
```diff
@@ -67,6 +67,18 @@
       "displayName": "INSECURE_REPOSITORY",
       "description": "Mark the image stream as an insecure repository",
       "value": "false"
+    },
+    {
+      "name": "MEMORY_MAX",
+      "displayName": "MEMORY_MAX",
+      "description": "Maximum memory for this app to have",
+      "value": "1000Mi"
```
Comment (cdowin) [inline omar/openshift/templates/apps/omar-oms-app.json]: I think the "default" max memory for OMS should be higher. 4-8GB?

File: omar/openshift/templates/apps/omar-geoscript-app.json [hunk 2]
```diff
@@ -67,6 +67,18 @@
       "displayName": "INSECURE_REPOSITORY",
       "description": "Mark the image stream as an insecure repository",
       "value": "false"
+    },
+    {
+      "name": "MEMORY_MAX",
+      "displayName": "MEMORY_MAX",
+      "description": "Maximum memory for this app to have",
+      "value": "1000Mi"
```
Comment (cdowin) [inline omar/openshift/templates/apps/omar-geoscript-app.json]: Higher default? 4-8Gb?

File: omar/openshift/templates/apps/omar-sqs-stager-app.json [hunk 3]
```diff
@@ -67,6 +67,18 @@
       "displayName": "INSECURE_REPOSITORY",
       "description": "Mark the image stream as an insecure repository",
       "value": "false"
+    },
+    {
+      "name": "MEMORY_MAX",
+      "displayName": "MEMORY_MAX",
+      "description": "Maximum memory for this app to have",
+      "value": "1000Mi"
```
Comment (cdowin) [inline omar/openshift/templates/apps/omar-sqs-stager-app.json]: Another higher default?

Comment (anaga670) [reply omar/openshift/templates/apps/omar-sqs-stager-app.json]: These are just the basic defaults that get overwritten in the deployConfig.yml

**Pull request merged**